File: src/main/java/com/sneakershop/backend/entity/order/enums/OrderStatus.java
package com.sneakershop.backend.entity.order.enums;

public enum OrderStatus {
    NEW,        // Mới
    PROCESSING, // Đang xử lý
    SHIPPING,   // Đang giao
    COMPLETED,  // Hoàn tất giao hàng
    PARTIALLY_RETURNED, // Đã hoàn trả một phần
    RETURNED,    // Đã hoàn trả toàn bộ
    CANCELLED   // Hủy
}

File: src/main/java/com/sneakershop/backend/entity/order/enums/OrderStatus.kt
package com.sneakershop.backend.entity.order.enums

enum class OrderStatus {
    NEW,  // Mới
    PROCESSING,  // Đang xử lý
    SHIPPING,  // Đang giao
    COMPLETED,  // Hoàn tất giao hàng
    CANCELLED // Hủy
}